components/molecules/FavoriteRepoCard/favorite-repo-card.tsx

```diff
@@ -0,0 +1,43 @@
+import Avatar from "components/atoms/Avatar/avatar";
+import { truncateString } from "lib/utils/truncate-string";
+import { StaticImageData } from "next/image";
+import Link from "next/link";
+import React from "react";
+
+export interface FavoriteRepoCardProps {
+  avatarURL?: string | StaticImageData;
+  name?: string;
+  handle?: string;
```

Comment:
`handle` is ambiguous with the verb `handle` - if it is referring to the data structure please refer to the `DbRepo` entity or when in doubt, check against the github api resource, for example checking https://api.github.com/repos/open-sauced/insights would suggest: `owner`, `username`, `login`.

Mention this because there is another prop `user` which doesn't fit any of the user data structures we have in the application. Please re-use the existing models, as the API and data structures are already defines, they can just be copied over 🍕

Reply:
I copied the names from Table Repository Names component, but yeah I agree with you

Reply:
Changed `handle` to `owner` as it follows the data structure. The `topic` and `user` are not native to any of the API or data structures tho, so I just renamed `user` to `userPage` since it is more indicative to its purpose. Should open an issue to have the same renames done in Table Repository Names for consistency.